patches/babel-plugin-typescript-to-proptypes+1.4.2.patch

```diff
@@ -1,29 +0,0 @@
-diff --git a/node_modules/babel-plugin-typescript-to-proptypes/lib/convertBabelToPropTypes.js b/node_modules/babel-plugin-typescript-to-proptypes/lib/convertBabelToPropTypes.js
-index 4d3252f..c939b42 100644
---- a/node_modules/babel-plugin-typescript-to-proptypes/lib/convertBabelToPropTypes.js
-+++ b/node_modules/babel-plugin-typescript-to-proptypes/lib/convertBabelToPropTypes.js
-@@ -141,6 +141,13 @@ function convert(type, state, depth) {
-         }
-         else if (NATIVE_BUILT_INS.includes(name)) {
-             return propTypes_1.createCall(core_1.types.identifier('instanceOf'), [core_1.types.identifier(name)], propTypesImportedName);
-+            // PATCH: Record<string, string> -> PropTypes.objectOf(PropTypes.string)
-+        }
-+        else if (name === 'Record') {
-+            const result = convert(type.typeParameters.params[1], state, depth);
-+            if (result) {
-+                return propTypes_1.createCall(core_1.types.identifier('objectOf'), [result], propTypesImportedName);
-+            }
-             // inline references
-         }
-         else if (state.referenceTypes[name]) {
-@@ -154,8 +161,8 @@ function convert(type, state, depth) {
-         else if (state.typeChecker) {
-             return convertTSToPropTypes_1.convertSymbolFromSource(state.filePath, name, state);
-         }
--        // Nothing found, so just omit
--        return null;
-+        // PATCH: return `any` in case a type cannot be mapped. For example, when the prop refers to a generic type.
-+        return propTypes_1.createMember(core_1.types.identifier('any'), propTypesImportedName);
```

Comment:
This functionality should be enabled now using  `mapUnknownReferenceTypesToAny: true`